controllers/past-grants/views/index.njk

```diff
@@ -42,34 +42,25 @@
             </div>
 
             <form class="u-inner" id="js-past-grants" method="get" action="" v-on:submit.prevent>
-                {# TODO: Improve this #}
-                <span>
-                    {% if queryParams.recipient %}
-                        <small v-show="filters.recipient">(Recipient: {{ grants[0].recipientOrganization[0].name }})</small>
-                    {% endif %}
-                    {% if queryParams.orgType %}
-                        <small v-show="filters.orgType">(Organisation Type: {{ grants[0].recipientOrganization[0].organisationType }})</small>
-                    {% endif %}
-                </span>
 
                 <div class="grants-searchbar">
                     <label class="ff-label" for="search-query">Search awarded grants</label>
                     <div class="form-field-addon u-margin-bottom-s">
                         <input class="form-field-addon__input js-only"
-                            type="search"
-                            id="search-query"
-                            name="q"
-                            :value="filters.q"
-                            autocomplete="no"
-                            @input="debounceQuery">
-
+                               disabled="disabled"
+                               type="search"
+                               id="search-query"
+                               name="q"
+                               :value="filters.q"
+                               autocomplete="off"
```

Comment:
Weirdly "no" seems to have changed and shows suggestions (at least, for me) so `off` stops it again.